source/reference/configuration.txt

```diff
@@ -12,16 +12,21 @@ Configuration
    :depth: 2
    :class: singlecol
 
-You can configure :program:`mongosync` instances at startup using a configuration
-file.  The configuration file contains settings that are the equivalent of
-``mongosync`` command-line options.
+You can configure :program:`mongosync` instances at startup using a
+configuration file.  The configuration file contains settings that are
+the equivalent of ``mongosync`` command-line options.
 
+The ``start`` command provides additional options for features like
```

Comment:
[optional consideration]

You could maybe make this a note at the top of the Options section along the lines of:

"Not all options can be set in the configuration file. The ``start`` command provides additional options for features like..."

This might make it a bit clearer that you may need to use both the config file and API options.